staging/src/k8s.io/cloud-provider/config/v1alpha1/types.go

```diff
@@ -98,4 +99,15 @@ type WebhookConfiguration struct {
 	// '-foo' means "disable 'foo'"
 	// first item for a particular name wins
 	Webhooks []string
+	// WebhookPort is the port that the controller-manager's webhook service runs on.
+	WebhookPort int32
+	// WebhookAddress is the IP address to serve on.
+	WebhookAddress string
+	// CaBundle is the ca certs to be used by apiserver for validating the webhook server certs.
+	CaBundle string
+	// ValidatingWebhookConfiguration is the config used for creating validation webhook config object.
+	ValidatingWebhookConfiguration *admissionregistrationv1.ValidatingWebhookConfiguration
```

Comment:
I'm very confused about what is going on here... why are we expecting to apply defaulting from REST APIs to fields of this config file?